flink-runtime/src/main/java/org/apache/flink/runtime/operators/util/BloomFilter.java

```diff
@@ -18,10 +18,10 @@
 
 package org.apache.flink.runtime.operators.util;
 
-import com.google.common.base.Preconditions;
 import org.apache.flink.core.memory.MemorySegment;
+import org.apache.flink.util.Preconditions;
```

Comment:
This import can be removed if the two `Preconditions.checkArgument` calls in the code are converted into `checkArgument` calls